CHANGELOG.md

```diff
@@ -81,7 +83,8 @@ as this was not intended to be used in the public API.
 
 This is the initial release to crates.io.
 
-[Unreleased]: https://github.com/rust-embedded-community/usb-device/compare/v0.3.1...HEAD
+[Unreleased]: https://github.com/rust-embedded-community/usb-device/compare/v0.3.2...HEAD
+[0.3.2]: https://github.com/rust-embedded-community/usb-device/compare/v0.3.0...v0.3.2
```

Comment:
cargo-release does this. It is actually [very simple to set up](https://github.com/eldruin/eeprom24x-rs/commit/bb4b4c9e7c88f2214de25d1f218e8c250fbc08fc) and does all the tagging, publishing and pushing for you. Per default it only does a dry run and shows you the diff of what it would do so you still see exactly what happens beforehand.

Reply:
I'll try that in the future, thanks for the recommendation :)